kitt-service.yml

```diff
@@ -35,6 +24,30 @@ instances:
               - --region={{$.kittExec.currentCluster.site}}
               - --zone={{$.kittExec.currentCluster.provider}}
   "02":
```

Comment:
Added a new lr instance for crud specific load only so that we can easily tune the crud load separately from the read loads.